source/brailleDisplayDrivers/eurobraille.py

```diff
@@ -188,7 +196,9 @@ def __init__(self, port="Auto"):
 				# Request device identification
 				self._sendPacket(EB_SYSTEM, EB_SYSTEM_IDENTITY)
 				# Make sure visualisation packets are disabled, as we ignore them anyway.
-				self._sendPacket(EB_VISU, EB_VISU_DOT, '0')
+				# py3 review: this is currently a zero char (0b110001 not 0b0).

```

Comment:
The eurobraille protocol uses real number characters as boolean values, so 0 (0x30) and 1 (0x31)